includes/class-woocommerce.php

```diff
@@ -0,0 +1,655 @@
+<?php
+/**
+ * Main WooCommerce Class
```

Comment:
We can change what this say, but the best practice is have a docblock for the file and another for the class.